qgis_plugin/geoai/core/venv_manager.py

```
@@ -2948,13 +3080,11 @@ def install_dependencies(
                         or "Return code {}".format(result.returncode)
                     )
                     _log(
-                        "Failed to install {}: {}".format(
-                            package_spec, error_msg[:500]
-                        ),
+                        "Failed to install {}: {}".format(package_spec, error_msg),
```

Ruff (0.15.21)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)